### DISCUSSION TITLE
Visual bug when doing a task like woodcutting or any skill

### DISCUSSION THREAD

**Discussion by SERdelicious** [runelite/runelite#18600] [Q&A]
![Desktop Screenshot 2024 12 10 - 17 47 09 24](https://github.com/user-attachments/assets/3c63d205-badd-47fd-998c-9756f347f54d)

**Comment by DominickCobb-rs** [comment]
that would be a plugin you've installed.

**Reply by SERdelicious** [reply]
i have had to delete and reinstall RL and it still happens

**Reply by SERdelicious** [reply]
any idea which plugin can cause this

**Reply by Felanbird** [reply]
We don't delete those settings on uninstall, otherwise people would be very upset their settings didn't persist through their "6 month break".

Click the 🔧 on the sidebar, then the 🔌 and uninstall whatever plugin you've installed, for this "feature".

**Reply by cdfisher** [reply]
This might be "Animation skipper".

Reinstalling RuneLite doesn't remove plugins you've installed so we wouldn't expect that to stop you from experiencing this.

**Reply by SERdelicious** [reply]
okay i will give that a try and see if it fixes the issue thanks!

**Reply by cdfisher** [reply]
I just looked at the source for that plugin, it's definitely Animation Skipper doing this

**Reply by SERdelicious** [reply]
it was animation skipper that was causing the the issue well thank for the help everyone really appreciate that :D
